### ISSUE TITLE
Update MemCached .net core support

### ISSUE THREAD

**Issue by ghost** [MichaCo/CacheManager#189]
The last commit for the current client used for MemCached was over 1 year ago. This repo is not updated for .net core support. However, there is another updated version (https://github.com/cnblogs/EnyimMemcachedCore) which compiles to .net standard 2.0.

Is it possible for you to use the updated version to allow for .net core support in combination with memcached? I gave it a look, only problem I found so far is that the mecached client uses .net standard 2.0 and cachemanager does not (yet).

Let me know what the possibilities are

**Comment by MichaCo** [comment]
Yup I will probably take a look at that after targeting testing/migrating to netstandard 2.0

**Comment by aqbjteddy** [comment]
@MichaCo How's going on upgrading work?

**Comment by MichaCo** [comment]
@aqbjteddy Hey. I haven't started, had a lot of work to do at work and currently I'm on vacation. 
Will do it when I find some time the next couple of weeks.

**Comment by MichaCo** [comment]
If there is still interest in adding netstandard2 and net50 target for the memcached package, I'd love to see a PR for this.
There are breaking changes between the 2 versions from Enyim, so I guess there will be a lot of if defs.

My only requirement would be that both implementations stay in the same nuget and just add additional targets using the new nuget.
And, who ever does the implementation also tests it. Because I don't really use memcached and cannot really test it.
#up-for-grabs

**Comment by MichaCo** [comment]
/closing - inactive - not going to fix 
Dropped memcached from V2
